includes/class-wc-calypso-bridge-setup.php

```diff
@@ -166,6 +166,25 @@ public function is_theme_installed( $theme ) {
 	public function register_payments_welcome_page() {
 		global $menu;
 
+		// WooCommerce must be active.
+		if ( !is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
+			return;
+		}
+
+		// WC Payment must not be installed.
+		if ( array_key_exists( 'woocommerce-payments/woocommerce-payments.php', get_plugins() ) ) {
```

Comment:
Should we just check if the plugin is active instead? We can ask for another chance by rendering this menu to those who installed the plugin then deactivated it.

Reply:
Good idea, yeah I think we should just check if it is active.

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/703/commits/4059fe445ca392e315163040c95a8ee6e19c1f98